app-misc/elasticsearch/elasticsearch-6.5.2.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
why not 7?

Reply:
IIRC, some of the inherited eclass wasn't compatible with it. But i'll retest tomorrow.